magma/t.py

```diff
@@ -73,6 +73,7 @@ def __le__(self, other):
         else:
             raise TypeError(f"Cannot use <= to assign to output: {self.debug_name} (trying to assign {other.debug_name})")
```

Comment:
Should we update this error message? (And maybe add a deprecation warning? Or maybe wait to do this)

Reply:
Heh was going to add another commit to address the error message but then decided I didn't actually want to write new code.   I will fix.

Deprecation warning is up to you.

Reply:
I'll add it as part of the new release after hwtypes2 is merged